include/theme/top_menu.js

```diff
@@ -92,10 +95,17 @@ module.exports = function TopMenuServiceModule(pb) {
      * @method getAccountButtons
      * @param {Object}   session
      * @param {Object}   ls      The localization service
+     * @param {String}   site    The current site
```

Comment:
Need to make it ```@param {String} [site] The current site```
The square brackets tell the documentation parser that the parameter is optional